docs/src/content/concepts-modes.md

```diff
@@ -138,6 +139,62 @@ most cases, the configuration will look like this:
 
 {{< figure src="/schematics/proxy-modes-transparent-3.png" >}}
 
+## WireGuard (transparent proxy)
+
+```shell
+mitmdump --mode wireguard
+```
+
+The WireGuard mode works in the same way as transparent mode, except that setup
+and routing client traffic to mitmproxy are different. In this mode, mitmproxy
+runs an internal WireGuard server, which devices can be connected to by using
+standard WireGuard clients applications, and just by importing the WireGuard
+configuration provided by mitmproxy.
```

Comment:
```suggestion
standard WireGuard clients applications:

1. Start `mitmweb --mode wireguard`.
2. Install Wireguard on target device.
3. Import the WireGuard config provided by mitmproxy on the device.
```

(documentation consumers generally hate prose and prefer simple lists...)

Reply:
LGTM.